packages/global-styles/src/components/global-styles-variations/index.tsx

```diff
@@ -146,7 +146,20 @@ const GlobalStylesVariations = ( {
 					} ) }
 				>
 					<div className="global-styles-variations__header">
-						<h2>{ headerText }</h2>
+						<h2>
+							{ ! splitDefaultVariation ? (
+								<>
+									<span>{ headerText }</span>
+									<PremiumBadge
```

Comment:
We can just hide the badge when the value of `splitDefaultVariation` is false.

```suggestion
									{ ! splitDefaultVariation && <PremiumBadge
```

Reply:
Sorry...the suggestion is a partial change 🤣 It should be

```js
<h2>
  <span>{ headerText }</span>
  { ! splitDefaultVariation && (
    <PremiumBadge
      shouldHideTooltip
      shouldCompactWithAnimation
      labelText={ translate( 'Included in your plan' ) }
    />
  ) }
</h2>
```